Runtimes/Supplemental/Readme.md

```diff
@@ -0,0 +1,52 @@
+# Swift Supplemental Libraries
+
+The supplemental libraries are all libraries that are not one of the Core or
+overlay libraries. Each supplemental library builds as an independent project.
+
+The supplemental libraries are:
+ - CxxInterop
+ - Differentiation
+ - Distributed
+ - Observation
+ - StringProcessing
+ - SwiftRuntime
```

Comment:
I think we should use the user visible name, not the one we need to specify in the CMake code
```suggestion
 - Runtime
```

Reply:
I assume that `SwiftRuntime` is actually `Backtracing`? If so, I think that we should clarify this as there is a SwiftRuntime in core as well.

Reply:
I think @al45tair was planning on exposing more of the lower-level runtime code to Swift through this module, not just backtracing. This is referring to the [RuntimeModule](https://github.com/swiftlang/swift/tree/main/stdlib/public/RuntimeModule) portion of the stdlib though.

Reply:
Yes. There are other things that I (and others) would like to put into the `Runtime` module, which is why it is not just called `Backtracing`.